modules/cluster-logging-must-gather.adoc

```diff
@@ -0,0 +1,170 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="gathering-cluster-logging-data_{context}"]
+= Gathering cluster logging data
+
+You can use the `cluster-logging-must-gather` tool to gather data that is specifically related to cluster logging and the Cluster Logging Operator in your {product-title} cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+
+* The OpenShift Container Platform CLI (oc) installed.
+
+.Procedure
+
+* Run the following `must-gather` command to gather cluster logging data:
+
+[source,terminal]
+----
+oc adm must-gather --image=quay.io/openshift/origin-cluster-logging-operator -- /usr/bin/gather
```

Comment:
`$` prompt

Reply:
Also updating the command per feedback from @mburke5678.